### PR TITLE
Fix doc for the VisualScript class after #39649 PR

### PR DESCRIPTION
**Pull request by swarnimarun**
Fixed the docs that I missed in the PR(https://github.com/godotengine/godot/pull/39649) and re-did the descriptions slightly to be more accurate.

### CODE REVIEW

File: modules/visual_script/doc_classes/VisualScript.xml [hunk 1]
```diff
@@ -26,23 +26,23 @@
 			</return>
 			<argument index="0" name="name" type="StringName">
 			</argument>
+			<argument index="1" name="arg1" type="int">
```
Comment (fire) [inline modules/visual_script/doc_classes/VisualScript.xml:29]: arg1 is invalid

**Review by fire — CHANGES_REQUESTED** [review]
Looks good to me except for the missing function argument.

File: modules/visual_script/doc_classes/VisualScript.xml (hunk 1 above)
Comment (akien-mga) [reply modules/visual_script/doc_classes/VisualScript.xml:29]: Yes, that means that you didn't give a name to the argument in the `bind_method` call.

File: modules/visual_script/doc_classes/VisualScript.xml [hunk 2]
```diff
@@ -26,23 +26,24 @@
 			</return>
 			<argument index="0" name="name" type="StringName">
 			</argument>
+			<argument index="1" name="func_node_id" type="int">
```
Comment (akien-mga) [inline modules/visual_script/doc_classes/VisualScript.xml:29]: You need to set the name in the C++ `bind_method`. This field is generated and will be overridden with `arg1` again.

File: modules/visual_script/doc_classes/VisualScript.xml [hunk 3]
```diff
@@ -26,23 +26,24 @@
...
+			</argument>
 			<description>
 				Add a function with the specified name to the VisualScript.
+				And assign the root VisualScriptFunction node's [code]node_id[/code].
```
Comment (akien-mga) [inline modules/visual_script/doc_classes/VisualScript.xml:33]:
```suggestion
				Add a function with the specified name to the VisualScript, and assign the root [VisualScriptFunction] node's [code]func_node_id[/code].
```

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**